haskell/sorting-in-reverse-order.md

```diff
@@ -0,0 +1,24 @@
+# Sorting in reverse order
+
+`Data.Ord` contains a newtype, `Down`, which can wrap any type that has an
+`Ord` instance, in order to provide a reversed `Ord` instance for that type.
+
+For example:
+
+```hs
```

Comment:
@gabebw `hs` is a valid language identifier. You can see what it looks like by checking the rendered diff:

![til 2015-01-27 13-56-11](https://cloud.githubusercontent.com/assets/1006966/5924750/41a7ce6c-a62c-11e4-885b-c33768414f73.jpg)